lambda_backend/route_microservice/src/deleteRoute.ts

```diff
@@ -1,18 +1,16 @@
 import { Handler } from 'aws-lambda';
 import DynamoDB, { AttributeValue, DeleteItemInput } from 'aws-sdk/clients/dynamodb';
-import S3, { DeleteObjectRequest } from 'aws-sdk/clients/s3';
 import jwt_decode from 'jwt-decode';
 import createError from 'http-errors';
-import { createHash } from 'crypto';
 
 import { getMiddlewareAddedHandler } from './common/middleware';
 import { getItemFromRouteTable } from './common/db';
 import { deleteRouteSchema } from './common/schema';
 import { DeleteRouteEvent, JwtPayload } from './common/types';
 import { getCognitoUserDetails } from './common/cognito';
+import { deleteRouteContentS3 } from './common/s3';
```

Comment:
Naming is a little inconsistent, why does this function have S3 behind it while `restoreRouteURL` doesn't?

Reply:
Can consider just removing the S3 from the function name + putting the utility functions like restore/trimRouteURL under s3/utils or something